pkg/monitoring/rules/alerts/operator.go

```diff
@@ -75,9 +75,9 @@ func operatorAlerts() []promv1.Rule {
 		},
 		{
 			Alert: "VMStorageClassWarning",
-			Expr:  intstr.FromString("(count(kubevirt_ssp_vm_rbd_block_volume_without_rxbounce > 0) or vector(0)) > 0"),
+			Expr:  intstr.FromString("kubevirt_ssp_vm_rbd_block_volume_without_rxbounce * on(name, namespace) kubevirt_vmi_info{os=\"Microsoft Windows\"} > 0"),
 			Annotations: map[string]string{
-				"summary":     "{{ $value }} Virtual Machines may cause reports of bad crc/signature errors due to certain I/O patterns.",
+				"summary":     "{{ $labels.namespace }}/{{ $labels.name }} Virtual Machine may cause reports of bad crc/signature errors due to certain I/O patterns.",
 				"description": "When running VMs using ODF storage with 'rbd' mounter or 'rbd.csi.ceph.com provisioner', VMs may cause reports of bad crc/signature errors due to certain I/O patterns. Cluster performance can be severely degraded if the number of re-transmissions due to crc errors causes network saturation.",
```

Comment:
Adjust the description so it states that only Windows VMs are considered?

Reply:
updated